app/models/force/application.rb

```diff
@@ -78,6 +78,7 @@ class Application < Force::ObjectBase
       { custom_api: 'processingStatus', domain: 'processing_status', salesforce: 'Processing_Status' },
       { custom_api: 'subStatus', domain: 'sub_status', salesforce: 'Sub_Status' },
       { custom_api: '', domain: 'status_last_updated', salesforce: 'Processing_Date_Updated' },
+      { custom_api: '', domain: 'supp_app_signed_date', salesforce: 'Supp_App_Signed_Date', type: 'date' },
```

Comment:
since you're converting this to customApi in line 189, this should have custom_api: 'suppAppSignedDate', right?